db/schema.rb

```diff
@@ -15,13 +15,13 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-  create_table "collections", force: :cascade do |t|
+  create_table "collections", id: :serial, force: :cascade do |t|
```

Comment:
many of us had a talk in person about this change, we feel it is safe to make as we don't think we will need more then 2^64 indexes.

Reply:
And while this is a change to the file, this should not represent a change to the db because serial was the default behavior before rails 5.X.  I think this is being explicitly stated as serial because the new default is BIGSERIAL in postgres.